client/verta/verta/external/cpython/LICENSE

```diff
@@ -0,0 +1,279 @@
+A. HISTORY OF THE SOFTWARE
```

Comment:
Everything in `client/verta/verta/external/cpython/` is vendored from Python 3.11. We specifically want the `graphlib` standard library.